src/main/kotlin/NewArchiveWindow.kt

```diff
@@ -0,0 +1,14 @@
+class NewArchiveWindow {
+    fun crateNewArchive (nameSet : Set<String>) {
+        println("Введите уникальное имя нового архива  или оставьте поле пустым для возврата в меню:")
+        val paragraph = scannerString(nameSet)                          // Ввод имени нового архива
+        if (paragraph.isNotEmpty()) {                                     // Если массив не пустой
+            val newMap: MutableMap<String, String> = mutableMapOf()     // Создаем пустой массив заметок для нового архива
+            dataList[paragraph] = newMap                             // Добавляем новый архив
```

Comment:
```suggestion
            dataList[paragraph] = mutableMapOf<String, String>()// Добавляем новый архив
```
Переменную незачем объявлять